pyproject.toml

```diff
@@ -44,3 +44,6 @@ valid-metaclass-classmethod-first-arg = "mcs"
 pythonpath = [
   "SentinelHub"
 ]
+markers = [
+    "qgis_integration: marks tests with qgis API interaction.",
```

Comment:
i see you've added a marker here. Did the marker approach not go through?

Reply:
yes, it failed to skip the tests at module level, but it did it locally, maybe I missed something, I will try it again in another PR